### DISCUSSION TITLE
Please tell me your favorite games

### DISCUSSION THREAD

**Discussion by ghost** [NAIST-SE/AttractingContributors#124] [General]
Please tell me a our favorite games
If you help me I will help too :-)

https://github.com/yudachi55/AttractingContributors-games

**Comment by nagata-atsuki** [comment]
added
Please help me
https://github.com/nagata-atsuki/AttractingContributors-Athlete

**Comment by veekay7** [comment]
Hey there, just added to your repo.
Please help.
https://github.com/VantelinLuca/AttractingContributors-WishCountry
